package.json

```diff
@@ -37,17 +37,18 @@
   },
   "devDependencies": {
     "@opentelemetry/api": "^0.20.0",
-    "@opentelemetry/context-async-hooks": "^0.20.0",
-    "@opentelemetry/core": "^0.20.0",
+    "@opentelemetry/instrumentation": "^0.20.0",
+    "@opentelemetry/instrumentation-http": "^0.20.0",
+    "@opentelemetry/node": "^0.20.0",
     "@opentelemetry/tracing": "^0.20.0",
     "@types/node": "^14.14.36",
     "@typescript-eslint/eslint-plugin": "^4.20.0",
     "@typescript-eslint/parser": "^4.20.0",
     "eslint": "^7.13.0",
     "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-standard": "^5.0.0",
```

Comment:
Eslint was giving me a warning, apparently we only need `eslint-config-standard` now